platform/ios/MGLMapView_IBAdditions.m

```diff
@@ -0,0 +1,57 @@
+#import "MGLMapView_IBAdditions.h"
+
+#import <objc/runtime.h>
+
+#import "NSProcessInfo+MGLAdditions.h"
+
+@implementation MGLMapView (IBAdditions)
+
++ (NSSet *)keyPathsForValuesAffectingLatitude
+{
+    return [NSSet setWithObject:@"centerCoordinate"];
+}
+
+- (double)latitude
+{
+    return self.centerCoordinate.latitude;
+}
+
+- (void)setLatitude:(double)latitude
+{
+    NSNumber *pendingLongitude;
+    if ((pendingLongitude = objc_getAssociatedObject(self, @selector(longitude))))
+    {
+        objc_setAssociatedObject(self, @selector(latitude), nil, OBJC_ASSOCIATION_RETAIN_NONATOMIC);
```

Comment:
> This is why these properties are strictly IB-only. Maybe I should assert that the coordinate is still invalid before proceeding?

I think this might be a good idea, so that you never get someone trying to set these via KVC or something.